### PR TITLE
Change group name totranslated

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: GroupBundle/DataFixtures/MongoDB/LoadGroupData.php [hunk 1]
```diff
@@ -60,11 +78,16 @@ public function getOrder()
      *
      * @return Group
      */
-    protected function generateGroup($name, $siteNumber, $referenceName, $role = null)
+    protected function generateGroup($name, $enLabel, $frLabel, $siteNumber, $referenceName, $role = null)
```
Comment (itkg-alavieille) [inline GroupBundle/DataFixtures/MongoDB/LoadGroupData.php]: update comment method

File: BackofficeBundle/Form/Type/GroupType.php [hunk 2]
```diff
@@ -13,13 +15,19 @@
...
     /**
-     * @param string $groupClass
+     * @param string                            $groupClass
+     * @param TranslateValueInitializerListener $translateValueInitializer
      */
-    public function __construct($groupClass)
+    public function __construct(
+        $groupClass,
+        TranslateValueInitializerListener $translateValueInitializer
+    )
```
Comment (itkg-ngilain) [inline BackofficeBundle/Form/Type/GroupType.php]: To respect PSR, in this case the ) and the { should be on the same line

File: GroupBundle/Tests/Form/Type/OrchestraGroupTypeTest.php [hunk 3]
```diff
@@ -4,6 +4,8 @@
 
 use OpenOrchestra\GroupBundle\Form\Type\OrchestraGroupType;
 use Phake;
+use OpenOrchestra\BackofficeBundle\Model\GroupInterface;
+
```
Comment (itkg-ngilain) [inline GroupBundle/Tests/Form/Type/OrchestraGroupTypeTest.php]: useless empty line

**Comment by itkg-nanne** [comment]
@itkg-ngilain , @itkg-alavieille done.

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**